### ISSUE TITLE
Joystick input crashing Dandara

### ISSUE THREAD

**Issue by marcelopio** [clementgallet/libTAS#397]
When I make any joystick input, Dandara is crashing for me.
Keyboard and mouse input works without any problem

Here are the logs:

```
Attempt 1: Connected.
[libTAS f:0] Thread 23158 (main) OpenGL vendor: Mesa/X.org
[libTAS f:0] Thread 23158 (main) OpenGL renderer: llvmpipe (LLVM 11.0.1, 256 bits)
Found path: /home/<>/.local/share/Steam/steamapps/common/Dandara/Dandara.x86_64
Mono path[0] = '/home/<>/.local/share/Steam/steamapps/common/Dandara/Dandara_Data/Managed'
Mono config path = '/home/<>/.local/share/Steam/steamapps/common/Dandara/Dandara_Data/Mono/etc'
Preloaded 'libsteam_api.so'
Unable to preload the following plugins:
	ScreenSelector.so
	libsteam_api.so
	ScreenSelector.so
Display 0 '2460G5 24"': 1920x1080 (primary device).
Logging to /home/<>/.config/unity3d/Long Hat House/Dandara/Player.log
[libTAS f:1] Thread 23158 (main) Generate SDL event SDL_CONTROLLERDEVICEADDED with joy 0
[libTAS f:1] Thread 23158 (main) Generate SDL event SDL_JOYDEVICEADDED with joy 0
[libTAS f:5] Thread 23158 (main) SDL_IsGameController call with id 0
[libTAS f:5] Thread 23158 (main) SDL_GameControllerOpen call with id 0
[libTAS f:5] Thread 23158 (main) SDL_GameControllerGetJoystick call with id 0
[libTAS f:5] Thread 23158 (main) SDL_JoystickInstanceID call with joy 0
[libTAS f:5] Thread 23158 (main) SDL_IsGameController call with id 0
[libTAS f:6] Thread 23234  ioctl call on device 14
[libTAS f:6] Thread 23234  ioctl call on device 14
[libTAS f:7] Thread 23236  ioctl call on device 26
[libTAS f:7] Thread 23236  ioctl call on device 26
[libTAS f:61] Thread 23237  ioctl call on device 38
[libTAS f:62] Thread 23237  ioctl call on device 38
[libTAS f:146] Thread 23158 (main)    evdev device 0 detected
[libTAS f:146] Thread 23158 (main) ioctl call on device 35
[libTAS f:146] Thread 23158 (main) ioctl access to EVIOCGBIT for event 53 on fd 35
[libTAS f:146] Thread 23158 (main) ioctl call on device 35
[libTAS f:146] Thread 23158 (main) ioctl access to EVIOCGEFFECTS on fd 35
[libTAS f:146] Thread 23158 (main)    evdev device 0 detected
[libTAS f:146] Thread 23158 (main)    evdev device 0 detected
[libTAS f:146] Thread 23158 (main) ioctl call on device -1
[libTAS f:146] Thread 23158 (main) ioctl call on device -1
[libTAS f:181] Thread 23239  ioctl call on device 35
[libTAS f:182] Thread 23239  ioctl call on device 35
[libTAS f:284] Thread 23158 (main) Generate SDL event CONTROLLERBUTTONDOWN with button 0
[libTAS f:284] Thread 23158 (main) Generate SDL event JOYBUTTONDOWN with button 0
[libTAS f:284] Thread 23158 (main) Generate evdev event EV_KEY with button 0
[libTAS f:284] Thread 23158 (main) ERROR: ioctl(evdevfds[evnum].first.first, FIONREAD, &pipeSize) == 0 failed in write_evdev with error Bad file descriptor
```

**Comment by clementgallet** [comment]
This should have been fixed by dedd3e20dfaa76bb440409ce274ed106fc79e484. Can you test the [last build](https://ci.appveyor.com/project/clementgallet/libtas/build/artifacts)?

**Comment by marcelopio** [comment]
It's working! Thanks!
